src/Card/Card.js

```diff
@@ -85,17 +86,12 @@ const Image = ({ image, className }) => {
 Image.displayName = Image;
 Card.Image = Image;
 
-const Superhead = ({ children, className }) => (
-  <span
-    className={dcnb("su-font-sans su-font-bold su-type-0 su-mb-0", className)}
-  >
-    {children}
-  </span>
-);
 Superhead.displayName = Superhead;
 Card.Superhead = Superhead;
 
-const Headline = ({ children }) => <Heading>{children}</Heading>;
+const Headline = ({ children, ...props }) => (
+  <Heading {...props}>{children}</Heading>
```

Comment:
Added ...props into the Headling because we need them 😸  Now the level/size props should work as expected